test/e2e/testing-manifests/ingress/nginx/rc.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       terminationGracePeriodSeconds: 0
       containers:
-      - image: gcr.io/google_containers/nginx-ingress-controller:0.8.3
+      - image: gcr.io/google_containers/nginx-ingress-controller:0.9.0-beta.1
```

Comment:
that's where it's used. I'm asking about the source for the image - it looks like the image was pushed without merging it into the repo, which makes it un-reproducible.